pb/v1/service.proto

```diff
@@ -101,3 +114,37 @@ message DeriveExtendedKeyResponse {
   // This field is 32 bytes long.
   bytes chain_code = 3;
 }
+
+// AddressEncoding enumerates the list of all supported encoding formats, for
+// serializing addresses.
+//
+// It is agnostic of the chain parameters.
+enum AddressEncoding {
+  ADDRESS_ENCODING_UNSPECIFIED  = 0;  // Fallback value if unrecognized / unspecified
```

Comment:
Seems to be unused. This is used from request payload only. So request should be rejected from rpc layer when there is unrecognized Address encoding provided or from `default:` behavior. Do you have future implementation/scenario in mind ? Otherwise I would suggest to remove it.

Reply:
I actually think the same. But one possible scenario is when the client is requesting an encoding that has been removed by the gRPC server.

**Example:** we support BCH cashaddr encoding, and suddenly BCH ceases to exist. We remove the encoding on the server, but the client forgets to update the protobuf stubs. If we don't have an `ADDRESS_ENCODING_UNSPECIFIED` field, the server will fallback to `ADDRESS_ENCODING_P2PKH`, which is wrong and can result in sending funds to the wrong address.

Do you think we need to be so defensive here?